### PR TITLE
Removes labels from tree data generation

### PR DESCRIPTION
**Pull request by thunterdb**
Note: it builds upon #81 , which should be reviewed first.

### CODE REVIEW

**Comment by thunterdb** [comment]
It actually needs to be added at a different place. WIP.

**Comment by jkbradley** [comment]
LGTM, thanks for realizing this.  Not sure why tests seemed to pass locally.  Maybe it's just unclear to me how to check for whether things pass; is that documented anywhere, or is it just buried in logs which get deleted after the tests run?  (I recall I had to hack the test to make it die and not delete the logs before exiting.)

File: src/main/scala/configs/mllib-small.yaml [hunk 1]
```diff
@@ -60,7 +60,7 @@ benchmarks:
       numExamples: 100
       numTestExamples: 10
       depth: 3
-      numClasses: 4
+      numClasses: 2
```
Comment (thunterdb) [inline src/main/scala/configs/mllib-small.yaml]: GBT only handles binary features, it seems

Comment (jkbradley) [inline src/main/scala/configs/mllib-small.yaml]: Yep

**Comment by jkbradley** [comment]
Now it LGTM!  Though I have not tested it myself

**Pull request merged**